src/main/resources/application.yml

```diff
@@ -8,10 +8,17 @@ spring:
         url: jdbc:postgresql://${DB_HOST:localhost}:5432/fittracker
         password: password
         username: user
+    profiles:
+        active: local
```

Comment:
this shouldn't be here, you've probably added it by mistake during rebase

Reply:
That's right